src/components/TextStyleOptions/TextStyleOptions.tsx

```diff
@@ -1,62 +1,66 @@
 /* eslint-disable no-console */
-import { ActionGroup, Item } from "@adobe/react-spectrum";
-import { Text } from "@adobe/react-spectrum";
-import type { Selection } from "@react-types/shared";
+import { Flex, ToggleButton } from "@adobe/react-spectrum";
 import TagBold from "@spectrum-icons/workflow/TagBold";
 import TagItalic from "@spectrum-icons/workflow/TagItalic";
+import { ColorPicker } from "components/ColorPicker";
 import React, { FC } from "react";
 
 export interface TextStyleProps {
     isBold?: boolean;
     isItalics?: boolean;
-    onChange?: (selectedKeys: (string | number)[]) => void;
+    onChange?: (options: { isBold?: boolean; isItalics?: boolean }) => void;
 }
 
 const TextStyleOptions: FC<TextStyleProps> = ({ isBold, isItalics, onChange }) => {
-    const [selected, setSelected] = React.useState<Selection>(() => {
-        const initialValue: string[] = [];
+    const [selectedBold, setSelectedBold] = React.useState<boolean | undefined>(isBold);
+    const [selectedItalics, setSelectedItalics] = React.useState<boolean | undefined>(isItalics);
```

Comment:
Quitemos todos los `React.*` usages - en lugar deberiamos estar importando `useState` y `useEffect` de React 

```javascript
import React, { FC, useEffect, useState } from "react";
```

```suggestion
    const [selectedBold, setSelectedBold] = useState<boolean | undefined>(isBold);
    const [selectedItalics, setSelectedItalics] = useState<boolean | undefined>(isItalics);
```

Reply:
On first, use si los props no están definidos, el `state` es `undefined`, cambiemoslo a que solo acepte `boolean`, [este link debería ser useful](https://stackoverflow.com/questions/31155477/convert-truthy-or-falsy-to-an-explicit-boolean-i-e-to-true-or-false) 
```suggestion
    const [selectedBold, setSelectedBold] = React.useState<boolean>(isBold);
    const [selectedItalics, setSelectedItalics] = React.useState<boolean>(isItalics);
```

<img width="289" alt="image" src="https://user-images.githubusercontent.com/52969178/231620478-d40b7f28-e5bb-439d-9c1e-ad182cc53f3e.png">